packages/pds/src/sequencer/sequencer-leader.ts

```diff
@@ -0,0 +1,128 @@
+import { DisconnectError } from '@atproto/xrpc-server'
+import { jitter, wait } from '@atproto/common'
+import { Leader } from '../db/leader'
+import { seqLogger as log } from '../logger'
+import Database from '../db'
+
+export const SEQUENCER_LEADER_ID = 1100
+
+export class SequencerLeader {
+  // leader = new Leader(SQUENCER_LEADER_ID, this.db)
+  leader: Leader
+
+  destroyed = false
+  polling = false
+  queued = false
+  lastSeq: number
+
+  constructor(public db: Database) {
+    const lockId = this.db.getSchemaAdvisoryLockId(SEQUENCER_LEADER_ID)
```

Comment:
had to add this because these advisory locks were conflicting across tests

Reply:
We had to deal with this in the appview too and took a slightly different approach— checkout https://github.com/bluesky-social/atproto/blob/46e1f051e169f4a4608d18660ecb165d0cdbf5c8/packages/dev-env/src/bsky.ts#L42C1-L43